pkg/reconciler/volumeclaim/pvchandler.go

```diff
@@ -55,8 +55,9 @@ func NewPVCHandler(clientset clientset.Interface, logger *zap.SugaredLogger) Pvc
 // where claim-name is provided by the user in the volumeClaimTemplate, and owner-name is the name of the
 // resource with the volumeClaimTemplate declared, a PipelineRun or TaskRun. If the PVC did not exist, a new PVC
 // with that name is created with the provided OwnerReference.
-// This function is only called when Affinity Assistant is disabled.
-// When Affinity Assistant is enabled, the PersistentVolumeClaims will be created by the Affinity Assistant StatefulSet VolumeClaimTemplate instead.
+// This function is only called when the Affinity Assistant behavior is AffinityAssistantPerWorkspace or AffinityAssistantDisabled.
+// When the Affinity Assistant behavior is AffinityAssistantPerPipelineRun or AffinityAssistantPerPipelineRunWithIsolation,
+// the PersistentVolumeClaims will be created by the Affinity Assistant StatefulSet VolumeClaimTemplate instead.
```

Comment:
I'm not a big fan of docstrings that explain how a function is called. I think it would be sufficient to explain that it creates PVCs for workspaces with volumeClaimTemplates

Reply:
I have removed the usage docstring for now. Will further look into the explanation in the followup refactor PR